code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -125,7 +125,6 @@ GLOBAL_LIST_EMPTY(roundstart_station_closets)
 		"Science" = list("icon_state" = "rd"),
 		"Security" = list("icon_state" = "cap"),
 		"Mining" = list("icon_state" = "mining"),
-		"Virology" = list("icon_state" = "bio_viro"),
```

Comment:
Why?

Reply:
FMPOV a viro closet carries the world flavor of "the stuff in here is potentially dangerous" and I think that's better than a plain medical closet